src/offline/progs/getdata_lima_adjust_mod.F90

```diff
@@ -175,7 +180,7 @@ SUBROUTINE GETDATA_LIMA_ADJUST (NPROMA, NGPBLKS, NFLEVG, KRR, KSV, &
   READ (IFILE) KLON, KDUM, KLEV
   READ (IFILE) KRR, KSV
   READ (IFILE) CCONDENS, CLAMBDA3
-  READ (IFILE) OSUBG_COND, OSIGMAS, PTSTEP, PSIGQSAT
+  READ (IFILE) OSUBG_COND, OSIGMAS, PTSTEP
```

Comment:
To avoid having to generate the files again, we should keep the same reading, even if it means doing nothing with this variable. Alternatively, it can be used to fill the ZSIGQSAT_B array.